addons/account/static/src/components/duplicate_bill_warninng/x2many_buttons.js

```diff
@@ -0,0 +1,81 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+import { getCurrency } from "@web/core/currency"
+
+
+class X2ManyButtons extends Component {
+    static template = "account.X2ManyButtons";
+    static props = {};
+    setup() {
+        this.action = useService("action");
+        this.orm = useService("orm");
+    }
+
+    async discard() {
+        if (this.props.record.isInEdition) {
+            await this.props.record.discard();
+        }
+    }
+
+    async open_tree_and_discard() {
+        const ids = this.current_field.currentIds;
+        await this.discard();
+        this.action.doAction({
+            name: _t("Records"),
+            type: "ir.actions.act_window",
+            res_model: this.current_field.resModel,
+            views: [[false, "list"]],
+            view_mode: "list",
+            domain: [["id", "in", ids]],
+        });
+    }
+
+
+    async open_form_and_discard(id) {
+        await this.discard();
+        this.action.doAction({
+            type: "ir.actions.act_window",
+            res_model: this.current_field.resModel,
+            res_id: id,
+            views: [[false, "form"]],
+            view_mode: "form",
+        });
+    }
+
+    get records_count() {
+        return this.current_field.count;
+    }
+
+    get loop_record_count() {
+        return this.current_field.count > 3 ? 3 : this.current_field.count;
```

Comment:
why?

Reply:
to show only the first 3 i just didn't wanna do it in xml